oaknational/Oak-Web-Application
# feat: svg hr

## Description

- Changes Hr component to use brush stroke svg

progresses #732 

## How to test

1. Go to launch page and about board page
2. Click on _______
3. You should see _______



## Checklist
